SQL/user/count_by_email.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
```

Comment:
Isn't this always 0 or 1?

Reply:
Yes it is. It's just more easy to read than using another query (check_by_email) and using weird logics to get to this 0 or 1.

Reply:
Why is it weird to check the existence of a user using `find_by_email`? it's basically the same thing as here, except it doesn't need to count unnecessarily. Not following your reasoning here.

Reply:
The logic to check was happening in the app.py and also it was hard to read! I should refactor it.

Reply:
the funny part is I forgot to use this query ! :))))

Reply:
I removed it !